detectable/src/main/java/com/synopsys/integration/detectable/detectables/maven/cli/MavenCliExtractor.java

```diff
@@ -50,17 +50,17 @@ public MavenCliExtractor(DetectableExecutableRunner executableRunner, MavenCodeL
 
     //TODO: Limit 'extractors' to 'execute' and 'read', delegate all other work.
     public Extraction extract(File directory, ExecutableTarget mavenExe, MavenCliExtractorOptions mavenCliExtractorOptions) throws ExecutableFailedException {
-        String[] mavenCommand = mavenCliExtractorOptions.getMavenBuildCommand()
-                                    .map(cmd -> cmd.replace("dependency:tree", ""))
-                                    .map(String::trim)
-                                    .map(cmd -> cmd.split(" "))
-                                    .orElse(new String[] {});
+        List<String> commandArguments = mavenCliExtractorOptions.getMavenBuildCommandArguments()
```

Comment:
Would it make sense to do the parsing from string to list<string> here? This class uses the command argument parser? I'm personally sort of on the fence - on the one hand this class just wants a list of arguments to call the executable list so just take the list - but on the other hand it modifies and edits the arguments so shouldn't it do all the modification. Any particular argument for the way it's done?

Reply:
I was hesitant to introduce blackduck-common as a dependency for the detectable module.  I didn't have a strong sense of where it made the most sense to convert the string to a list either, so I let that be the tie breaker.

Reply:
Thats a pretty good reason 😄